laserscan_multi_merger:
  ros__parameters:
    destination_frame: "base_link"
    cloud_destination_topic: "/merged_cloud"
    scan_destination_topic: "/scan/multi"
    laserscan_topics: "/scan"
    angle_min: -3.14
    angle_max: 3.14
    angle_increment: 0.0058
    scan_time: 0.0333333
    range_min: 0.1
    range_max: 100.0
